frontend/src/components/TaskTable.tsx

```diff
@@ -161,10 +161,16 @@ export const TaskTableUnrated: React.FC<{
   useEffect(() => {
     const type = getType(userInfo?.roles, currentRoadmap?.id);
     if (type === RoleType.Admin) {
-      if (userInfo && currentRoadmap && allUsers)
+      if (userInfo && currentRoadmap && allUsers && tasks) {
+        // currentRoadmap.customers breaks taskUtils/unratedProductOwnerTasks if it's undefined
         setTaskList(
-          unratedProductOwnerTasks(tasks, allUsers, currentRoadmap.customers),
+          unratedProductOwnerTasks(
+            tasks,
+            allUsers,
+            currentRoadmap.customers ?? [],
```

Comment:
This is actual type problem 👍 

The comment could be attached to this line, and be more specific to the problem, e.g.
```suggestion
            currentRoadmap.customers ?? [], // FIXME: incorrect type, customers may be undefined
```

Similar comment could maybe be added to the corresponding type definition (in redux/roadmaps/types), until it gets fixed